third_party/terraform/utils/error_retry_predicates.go

```diff
@@ -33,6 +33,10 @@ func pubsubTopicProjectNotReady(err error) (bool, string) {
 
 func isSqlOperationInProgressError(err error) (bool, string) {
 	if gerr, ok := err.(*googleapi.Error); ok && gerr.Code == 409 {
+		if strings.Contains(gerr.Body, "you cannot reuse the name of the deleted instance until one week from the deletion date.") {
```

Comment:
hmmm, i don't like this, but I'm not sure if we have a better option

Reply:
I agree, I could use a smaller portion of the message, in case it were to change, but I'm not sure where to cut it off.  I wasn't sure what else to key off of, because it's only the message that differentiates it from other 409s